packages/viem/src/index.ts

```diff
@@ -0,0 +1,291 @@
+import { hashTypedData, serializeTransaction, signatureToHex } from "viem";
+import { toAccount } from "viem/accounts";
+import { keccak256 } from "viem";
+import type {
+  HashTypedDataParameters,
+  LocalAccount,
+  SerializeTransactionFn,
+  SignableMessage,
+  TransactionSerializable,
+  TypedData,
+} from "viem";
+import {
+  TurnkeyApi,
+  TurnkeyActivityError,
+  init as httpInit,
+} from "@turnkey/http";
+
+/**
+ * Type bundling configuration for an API Key Viem account creation
+ */
+type TApiKeyAccountConfig = {
+  /**
+   * Turnkey API public key
+   */
+  apiPublicKey: string;
+  /**
+   * Turnkey API private key
+   */
+  apiPrivateKey: string;
+  /**
+   * Turnkey API base URL
+   */
+  baseUrl: string;
+  /**
+   * Turnkey organization ID
+   */
+  organizationId: string;
+  /**
+   * Turnkey private key ID
+   */
+  privateKeyId: string;
+};
+
+/**
+ * Creates a new Custom Account backed by a Turnkey API key.
+ */
+export async function createApiKeyAccount(
+  config: TApiKeyAccountConfig
+): Promise<LocalAccount> {
+  const { apiPublicKey, apiPrivateKey, baseUrl, organizationId, privateKeyId } =
+    config;
+
+  httpInit({
```

Comment:
If we use this global init, and users create multiple api key accounts, are the newer instances going to clobber the configs of the older instances?